pkg/cmd/chat/user/user.go

```diff
@@ -296,3 +305,310 @@ func revokeCmd() *cobra.Command {
 
 	return cmd
 }
+
+func banCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ban-user --target-user-id [user-id] --banned-by-id [user-id] --reason [reason] --expiration [expiration-in-minutes]",
+		Short: "Ban a user",
+		Long: heredoc.Doc(`
+			Users can be banned from an app entirely.
+			When a user is banned, they will not be allowed to post messages until
+			the ban is removed or expired but will be able to connect to Chat
+			and to channels as before.
+
+			Channel watchers cannot be banned.
```

Comment:
I don't think this statement is true, where did you see this?

Reply:
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/19969687/164709749-a6525879-c711-4f68-8d03-617a0a9371a2.png">